tools/rem_complex/rem_complex.xml

```diff
@@ -23,7 +23,12 @@
         <param name="input" format="inchi,smi,csv" type="data" help="Accepted input formats: CSV, SMI, and InChI."/>
```

Comment:
Would be good to also support tsv as input

Reply:
done in [530212f](https://github.com/RECETOX/galaxytools/pull/504/commits/530212f6efa29c3aa625d93e557b2ae0535e988b)